.gitpod.dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM gitpod/workspace-base:2023-11-04-12-07-48
+
+USER root
+
+RUN apt-get update \
+  && apt-get install -y --no-install-recommends luarocks=3.8.0+dfsg1-1 \
+  && apt-get clean \
+  && rm -rf /var/lib/apt/lists/*
```

Comment:
This line removes the package lists. Why?

Reply:
It it reduces the image size, cf. [DL3009](https://github.com/hadolint/hadolint/wiki/DL3009).

Reply:
I'm not sure that the advice applies here. For a deployment setup (what docker is usually used for), you don't need the ability to install new packages, but a development setup shouldn't be "bricked" - devs may want to install further packages.

Reply:
They can still run `sudo apt-get update`.